src/features/partnership.feature

```diff
@@ -21,19 +22,19 @@ Feature: partnership
         And I go to a special QA page at url "/qa/mid-and-east-antrim/main-partnership-contact" with injected "two-partnership" data
         Then I expect that element "mainPartnershipContact.partnerOne" contains the text "one"
         And I expect that element "mainPartnershipContact.partnerTwo" contains the text "two"
-        When I click on the element "commonElements.continueButton"
-        Then I expect that element "mainPartnershipContact.error" contains the text "You must select the main partnership contact before continuing"
```

Comment:
why is this text no longer there?

Reply:
This is a mistake, i took it out becuase i wasnt sure where to put the error in the njk file and it wasnt returned. will fix asap